Automattic/sensei
# Add onboarding features installation

Fixes #5647
Based on https://github.com/Automattic/sensei/pull/5823

### Changes proposed in this Pull Request

* It adds the installation of the features (last step of the new onboarding).
* When it's completed, it makes the setup wizard as finished.

### Testing instructions


* Without Sensei Certificates and WooCommerce installed, go to the Setup Wizard.
* In the purpose step, select the 2 first items (both plugins should be installed).
* Navigate until the last step, and make sure both plugins are installed and activated.
* Repeat the test starting with the plugins installed, but deactivated.
* Make sure they are activated after completing the Setup Wizard.


### Deprecated Code

* `Sensei_Setup_Wizard::get_sensei_extensions`
* `Sensei_Setup_Wizard::get_feature_with_status` (a private method, but it's deprecated so the `get_sensei_extensions` continues working).
* `Sensei_REST_API_Setup_Wizard_Controller::complete_setup_wizard`


### Screenshot / Video

https://user-images.githubusercontent.com/876340/193364517-6aaa3639-cc04-4de0-abef-f6d8eca475e4.mov
